OpenRA.Mods.Common/Traits/Player/PlayerRadarTerrain.cs

```diff
@@ -88,8 +69,8 @@ public void WorldLoaded(World w, WorldRenderer wr)
 				foreach (var uv in world.Map.AllCells.MapCoords)
 					UpdateTerrainCellColor(uv);
 
-				world.Map.Tiles.CellEntryChanged += UpdateTerrainCell;
-				world.Map.CustomTerrain.CellEntryChanged += UpdateTerrainCell;
+				world.Map.Tiles.CellEntryChanged += cell => UpdateTerrainCell(cell.ToMPos(world.Map));
```

Comment:
`CellEntryChanged` may well be converting `MPos` to `CPos` before calling these.

We may want to consider introducing a `CellLayer.MapCellEntryChanged` in a future PR to reduce the number of conversions.